app/src/main/java/com/github/swent/echo/compose/components/ListDrawer.kt

```diff
@@ -25,261 +30,225 @@ import androidx.compose.runtime.mutableStateOf
-                            ),
-                        contentDescription = stringResource(id = R.string.list_drawer_event_status),
-                        tint = textColor,
+                Column {
+                    Text(
+                        text = event.title,
+                        modifier =
+                            Modifier.fillMaxWidth(0.8f)
+                                .testTag("list_event_title_${event.eventId}"),
+                        overflow = TextOverflow.Ellipsis,
+                        maxLines = 2,
+                        style = MaterialTheme.typography.titleLarge,
                     )
-                    Spacer(modifier = Modifier.width(5.dp))
+                    Spacer(modifier = Modifier.height(4.dp))
                     Text(
-                        text = "${event.participantCount}/${event.maxParticipants}",
-                        modifier = Modifier.testTag("list_event_participants_${event.eventId}"),
-                        textAlign = TextAlign.Center,
-                        color = textColor,
+                        text = "$association$dist$date",
+                        style = MaterialTheme.typography.bodySmall,
+                        modifier = Modifier.testTag("list_event_date_${event.eventId}"),
                     )
                 }
-                Spacer(modifier = Modifier.height(5.dp))
-                // Display event range (epfl, section, class, ...)
-                // Need to be improved to not find only "in" and "ba6" but everything (need to see
-                // Noah's implementation)
-                Box(
-                    modifier =
-                        Modifier.clip(RoundedCornerShape(25.dp))
-                            .height(25.dp)
-                            .width(60.dp)
-                            .background(
-                                if (event.tags.any { tag -> tag.name.lowercase() == "epfl" })
-                                    colorEpfl
-                                else if (
-                                    event.tags.any { tag ->
-                                        tag.name.lowercase() == section.lowercase()
-                                    }
-                                )
-                                    colorSection
-                                else if (
-                                    event.tags.any { tag ->
-                                        tag.name.lowercase() == semester.lowercase()
-                                    }
-                                )
-                                    colorClass
-                                else color2ndFloor
-                            )
+                Column(
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally,
                 ) {
+                    Icon(painterResource(R.drawable.full), contentDescription = "Group")
```

Comment:
This icon description could be more explicit so that the number at its right makes more sense when screen readers read it, e.g. "number of participants".